cluster/gce/gci/configure-helper.sh

```diff
@@ -366,9 +366,6 @@ function create-master-auth {
   if [[ -n "${KUBE_SCHEDULER_TOKEN:-}" ]]; then
     append_or_replace_prefixed_line "${known_tokens_csv}" "${KUBE_SCHEDULER_TOKEN},"          "system:kube-scheduler,uid:system:kube-scheduler"
   fi
-  if [[ -n "${KUBELET_TOKEN:-}" ]]; then
```

Comment:
@liggitt right now the tls bootstrap is implemented only in gce, gci, debian, and ubuntu. I'll take a quick scan of those. I'm planning on removing container-linux this release.